src/Components/NeedMilk/NeedMilk.js

```diff
@@ -1,30 +1,14 @@
 import './NeedMilk.scss';
-import '../DonorCard/DonorCard';
+import DonorCard from '../DonorCard/DonorCard';
 
 const NeedMilk = () => {
-  //let display = donors.map(donor => {
-  // <DonorCard
-  //  name={}
-  //  age={}
```

Comment:
When we put this into play, we won't need age anymore since we eliminated it in the schema. That's a later thing though.